shared/wallets/confirm-form/container.js

```diff
@@ -8,4 +8,5 @@ const mapDispatchToProps = (dispatch: Dispatch) => ({})
 
 const mergeProps = (stateProps, dispatchProps) => ({})
 
+// $FlowIssue TODO
```

Comment:
what's the issue here?

Reply:
these weren't implemented yet so we werent passing any props which is incorrect. cjbs pr fills this stuff in i believe